apps/web/app/[lang]/sponsors/components/About.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button, NativeH1, P1, P2 } from 'ui/core';
+import SemiCircleLeft from '@public/images/shared/semi-circle-left.png';
+import SemiCircleGray from '@public/images/shared/semi-circle-gray.png';
+import { ImageWithFallback } from '@components/utils/ImageWithFallback';
+import { useTranslations } from 'intl';
+
+export const About = () => {
+  const t = useTranslations('sponsors.about');
+
+  return (
+    <div className="w-full flex flex-col gap-[72px] pb-16">
+      <div className="w-full flex justify-between items-stretch p-4">
```

Comment:
```suggestion
      <div className="w-full flex justify-between items-stretch">
```

Remove that extra padding